samples/flip-coin/README.md

```diff
@@ -0,0 +1,54 @@
+# The flip-coin pipeline
+
+The [flip-coin pipeline](https://github.com/kubeflow/pipelines/blob/master/samples/core/condition/condition.py)
+is used to demonstrate the use of conditions.
+
+* [Installing Tekton Pipelines on Kubernetes](https://github.com/tektoncd/pipeline/blob/master/docs/install.md#installing-tekton-pipelines-on-kubernetes) 
+
+* [Installing Tekton Pipelines on OpenShift](https://github.com/tektoncd/pipeline/blob/master/docs/install.md#installing-tekton-pipelines-on-openshift)
+
+* [Configuring artifact storage](https://github.com/tektoncd/pipeline/blob/master/docs/install.md#configuring-artifact-storage): When using S3 bucket, make sure to update your bucket name.
+
+There are two ways to generate `Tekton` YAML. One is using `DSL-COMPILE-TEKTON` command line tool, another way is using `TektonCompiler` inside your pipeline script. Both will generate the same `Tekton` YAML file. Then apply the YAML to your cluster by using the `tkn` command.
```

Comment:
`dsl-compile-tekton` should be lower case

Reply:
done